docusaurus/docs/create-a-plugin/develop-a-plugin/panel-properties.md

```diff
@@ -0,0 +1,106 @@
+---
+id: subscribe-events
+title: Subscribe to Grafana events
+description: Work with the event bus to subscribe to Grafana application events
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - panel
+  - properties
+  - eventbus
+  - event bus
+  - subscribe
+  - events
+sidebar_position: 5
+---
+
+# Subscribe to Grafana events
+
+If you’re building a [panel plugin](../../introduction/plugin-types-usage.md#panel-visualization-plugins), Grafana already provides the data you need to configure and render your panel. If not, you can usually retrieve the information you need from any of the runtime services, like `BackendSrv` and `TemplateSrv`.
+
+However, in some cases you may want your plugin to react to changes that occur outside of your plugin. For example, you may want your plugin to react when the user hovers their cursor over data in another panel. In this guide, you’ll learn how to make your plugin react to events in Grafana.
```

Comment:
What if we make this text aligned with the example we offer down below, the zoom out out. I am not sure if we offer an event for hovering in another panel

Reply:
done.